### ISSUE TITLE
Cannot set interface up when certain interface used in OVS bridge which about to be deleted.

### ISSUE THREAD

**Issue by cathay4t** [nmstate/nmstate#779]
https://bugzilla.redhat.com/show_bug.cgi?id=1799059

**Comment by cathay4t** [comment]
Create OVS using eth1 with OVS internal interface.

```yml
---
interfaces:
- name: eth1
  type: ethernet
  state: up
- name: ovs-br0
  type: ovs-bridge
  state: up
  bridge:
    port:
    - name: ovs0
    - name: eth1
- name: ovs0
  type: ovs-interface
  state: up
  ipv4:
    address:https://bugzilla.redhat.com/show_bug.cgi?id=1799059
    - ip: 192.0.2.1
      prefix-length: 24
    dhcp: false
    enabled: true
  mtu: 1500
```

Remove bridge but keep eth1 up using the IP address of OVS interface:


```yml
---
interfaces:
- name: eth1
  type: ethernet
  state: up
  ipv4:
    address:
    - ip: 192.0.2.1
      prefix-length: 24
    dhcp: false
    enabled: true
- name: ovs-br0
  type: ovs-bridge
  state: absent
```

**Comment by EdDev** [comment]
Should be tested with #796 .

**Comment by tyll** [comment]
> Should be tested with #796 .

Since #796 is merged, can we close this?

**Comment by cathay4t** [comment]
@tyll The problem is blocked by #809 . But downstream bug should be fixed.

**Comment by cathay4t** [comment]
This is fixed NetworkManager 1.22.10.
